tests_e2e/tests/extensions_disabled/extensions_disabled.py

```diff
@@ -47,27 +48,43 @@ def run(self):
         output = ssh_client.run_command("update-waagent-conf Extensions.Enabled=n", use_sudo=True)
         log.info("Disable completed:\n%s", output)
 
-        # From now on, extensions will time out; set the timeout to the minimum allowed(15 minutes)
+        # Set the timeout to the minimum allowed(15 minutes) so we can check that the extension fails before timeout
```

Comment:
let's remove this. we were forcing the timeout to 15 min because the extensions were being ignored, now that they fail this is no longer needed